packages/react-query-devtools/src/devtools.tsx

```diff
@@ -277,6 +277,8 @@ export function ReactQueryDevtools({
         return () => {
           window.removeEventListener('resize', run)
           if (root?.parentElement && typeof previousValue === 'string') {
+            // unset padding for resetting
+            root.parentElement.style.padding = ''
```

Comment:
what if the parentElement had a padding before? Should we not store the old padding and re-use that one?

tbh, I think meddling with the parentElement is not a good idea because it could seriously destroy the layout of the app. Maybe there is a way to achieve what we want _without_ setting some styles on the parent element?

Reply:
You are right.
I will consider other approaches 👍

Reply:
@TkDodo 
I think the reason for the padding on the parent element is to prevent the element from wrapping around behind the devtools.
It seems difficult to solve this without affecting the parent element, so I think it is a trade-off 😇

For now, we have responded to restore the original padding.

The only other thing I can think of is...
 - Adding a opt-out / opt-in flag
 - Giving up that prevent wrapping

Any good ideas?

Reply:
reverting back to the previous style seems like a good idea 👍